<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="lLeft">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="lIso">
        <item>
         <widget class="QLabel" name="LabelIso">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Iso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="BoxIso"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lAperture">
        <item>
         <widget class="QLabel" name="LabelAperture">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Aperture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="BoxAperture"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lShutter">
        <item>
         <widget class="QLabel" name="LabelShutter">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Shutter Duration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="BoxShutter"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lColours">
        <item>
         <widget class="QListView" name="ListColours"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lColourControl">
          <item>
           <widget class="QPushButton" name="ButtonAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonUp">
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonDown">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="OpenGlBox" name="PreviewWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lAboveGo">
        <item>
         <layout class="QHBoxLayout" name="lFormat">
          <item>
           <widget class="QLabel" name="LabelProcessedFormat">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Processed Format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="BoxProcessedformat"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lSave">
          <item>
           <widget class="QCheckBox" name="CheckSaveProcessed">
            <property name="text">
             <string>Save Processed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CheckSaveRaw">
            <property name="text">
             <string>Save Raw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lDelay">
          <item>
           <widget class="QLabel" name="labelDelay">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SpinDelay"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonGo">
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGlBox</class>
   <extends>QOpenGLWidget</extends>
   <header>openglbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
